packages/graphql-language-service-interface/src/getAutocompleteSuggestions.ts

```diff
@@ -202,19 +203,22 @@ function getSuggestionsForFieldNames(
 function getSuggestionsForInputValues(
   token: ContextToken,
   typeInfo: AllTypeInfo,
-): Array<CompletionItem> {
+): CompletionItem[] {
   const namedInputType = getNamedType(typeInfo.inputType as GraphQLType);
   if (namedInputType instanceof GraphQLEnumType) {
-    const values = namedInputType.getValues();
+    const values: GraphQLEnumValues[] = namedInputType.getValues();
```

Comment:
`GraphQLEnumValues[]` is not defined in this file, and without defining a type here, `values` is implied as `GraphQLEnumValue[]` from `namedInputType.getValues()`. I'm wondering if it's just a mistake or is there any reason to define a type for `values` here?